Add team lead guide for onboarding a new CSM (#19330)
PostHog/posthog.com: contents/handbook/cs-and-onboarding/new-hire-onboarding-team-leads.md

---
title: Onboarding a new CSM (team leads)
sidebar: Handbook
showTitle: true
---

So you've got a new CSM joining your team! This page is for team leads - the joiner-facing plan is [new starter onboarding](/handbook/cs-and-onboarding/new-hire-onboarding). Here's what to prepare and when:

1. **Before they join** - prepare their book, book the 1:1s, plan the in-person week
2. **First week** - welcome 1:1, get the team sharing calls and tips
3. **In-person week** (usually week 2) - 4 days working together, agenda template below
4. **First month** - any spillover from in-person onboarding

## Before they join

**Prepare their book of business:** 

> - Accounts come from a few places: Simon pulls net-new accounts, teammates can offload some if they're overloaded, and product-led sales and new business sales can have accounts waiting for a CSM handover - shoot them an ask. 

Reviewer: ```suggestion
- Accounts come from a few places (listed by priority): CSMs can offload accounts if they're overloaded, product-led sales and new business sales can have accounts waiting for a CSM handover, and Simon frequently pulls net-new accounts - ask all groups to see what needs coverage. 
```

Reviewer: My own prioritization I've been using, curious if you disagree!

Author: agreed and good call out!

- Tee up ~20 accounts so they're not at capacity immediately - it also leaves room for late handovers. 
- Follow the [capacity model](/handbook/cs-and-onboarding/how-we-work), and bonus points if accounts can line up with their previous experience.

> **Book the 1:1s:** Welcome 1:1 on day 1, a recurring 1:1, and (optionally) an extended 1:1 every 4 weeks.
> 

Reviewer: ```suggestion
**Book the 1:1s:** Welcome 1:1 on day 1, a recurring 1:1, and (optionally) an extended 1:1 every 4 weeks.

**Find an onboarding buddy:** Another CSM (same geography) to schedule a 1:1 and provide additional support if they need it.
```

      - How: [what we value](https://posthog.com/handbook/values), [a wide company](https://posthog.com/handbook/wide-company) with [small teams](https://posthog.com/handbook/company/small-teams)
    - GTM / Customer Success:
      - [GTM overview](https://posthog.com/handbook/growth/sales/overview)
      - [CS overview](https://posthog.com/handbook/cs-and-onboarding/customer-success) + have a skim of the follow-on links
      - [Use case selling](https://posthog.com/handbook/growth/use-case-selling/use-case-selling)
      - [CSM + TAM](https://posthog.com/handbook/growth/sales/account-allocation)
      - [Contract rules](https://posthog.com/handbook/growth/sales/contract-rules)
- Work through your onboarding plan - make sure to prioritize Drata
- Practice your demo
- The plan for onboarding week
How we work
- Sprint planning issue on GitHub
- By default ask questions in #team-customer-success unless it needs to be private
Questions from me
- What's your GitHub handle so I can add you to the sprint template?
- What do I need to know about you to help you do your best work?
```

</details>

**Ask the team to share things!** And for calls the new CSM can shadow on.

> **Optionally, a check-in 1:1 on day 3.** The first week is a lot of solo reading - a midweek touchpoint helps.

Reviewer: ```suggestion
**Optionally, a check-in 1:1 on day 3.** The first week is a lot of solo reading - a midweek touchpoint helps. Also check to see if the onboarding buddy has scheduled a 1:1 as well!
```

## In-person week (usually week 2)

> [In-person onboarding](/handbook/people/onboarding#in-person-onboarding) covers the company-wide process. Below is how we run the CSM week - there's a [4-day agenda template](https://posthog.slack.com/docs/TSS5W8YQZ/F0BNBPN78EP) to start from, which we copy into a canvas in their onboarding channel. It's a living template - add what you learn from each onboarding.

Reviewer: It seems access to that canvas is locked! Might be good to make it open.

- **Finding info** - where to look (handbook vs repo vs Slack), maybe even a working exercise (e.g. "what is the Teams plan?")

#### Understanding a customer's usage (day 2)

- **What is the customer using PostHog for?** The products they pay for, the events they capture, what they do with the data
- **How's the relationship?** The active users and what they're doing (candidates for outreach), recent tickets and calls, which PostHog humans they've talked to (AE, onboarding, TAM, CSM)
- **Are they set up well and getting value?** Run the [basic implementation review](/handbook/cs-and-onboarding/foundation-check) and [health check](/handbook/cs-and-onboarding/health-checks)
- **Commercials:**
  - Monthly usage - forecasted MRR, billing limits, platform add-ons
  - The contract, if they're on one - how much credit they bought, when they renew, are they on track? Talk through [contract rules](/handbook/growth/sales/contract-rules) and the tools we use (QuoteHog, PandaDoc, Salesforce)

#### Prioritizing across the book + deciding next steps (days 3-4)

> - **Prioritizing a book** - the signals we look at and which ones are more worrisome, both when a book is first assigned and on an ongoing basis. Have them set up their own Vitally views

Reviewer: ```suggestion
- **Prioritizing a book** - the signals we look at and which ones are more worrisome, both when a book is first assigned and on an ongoing basis. Have them set up their own Customer Analytics views
```

Feel like we'd rather just get them going there, given the change is in a couple months!

- **Deciding next steps** - by now they should have a read on where each account's at (or know how to find out). Use next steps that haven't come up yet as chances to talk them through - e.g. a new Slack channel (invite you + support-hog), an upcoming renewal (how renewals work)

#### Throughout the week

> - Demo practice with feedback

Reviewer: ```suggestion
- Demo practice with feedback (the more team members attending, the better)
```
